### PR TITLE
Add systemd configs

### PR DESCRIPTION
**Pull request by mschnepf**
This pull request adds documentation and a sample configuration for use as a ``systemd`` service unit.

### CODE REVIEW

File: systemd/cobald@.service [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=COBalD - the Opportunistic Balancing Daemon for %I
+After=network.target
+
```
Comment (maxfischer2781) [inline systemd/cobald@.service]: Please add a section to allow enabling this service automatically:

```
[Install]
RequiredBy=multi-user.target
```

**Comment by maxfischer2781** [comment]
Make sure to add the docs page to ``cobald/docs/source/daemon/overview.rst`` as well, please.

File: docs/source/daemon/systemd.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
+===============
+systemd Configs
+===============
+
+You can run :py:mod:`cobald` as a system service.
+We provide systemd configs for multiple :py:mod:`cobald` instances run as services.
+You can manage several instances which are identified with a systemd instance name.
+
+Create a file named  ``cobald@.service`` into the ``/usr/lib/systemd/system`` directory.
```
Comment (eileen-kuehn) [inline docs/source/daemon/systemd.rst]:
```suggestion
Create a file named  ``cobald@.service`` in the ``/usr/lib/systemd/system`` directory.
```

File: docs/source/daemon/systemd.rst [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+An example of a systemd config file:
+
+.. literalinclude:: cobald@.service
+
+In this example, the configs for the different COBalD instances are located at ``/etc/cobald/instance-name.py``.
+:py:mod:`cobald` can handle ``.py`` and ``.yaml`` configuration files.
+Please ensure that the choosen python interpreter has :py:mod:`cobald` installed!
```
Comment (eileen-kuehn) [inline docs/source/daemon/systemd.rst]:
```suggestion
Please ensure that the chosen python interpreter has :py:mod:`cobald` installed!
```

File: docs/source/daemon/systemd.rst [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+We recomment to use a ``virtualenv``.
```
Comment (eileen-kuehn) [inline docs/source/daemon/systemd.rst]:
```suggestion
We recommend to use a ``virtualenv``.
```

File: docs/source/daemon/systemd.rst [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+        $ systemctl stop cobald@instance-name
+
+ -  report the current status of the :py:mod:`cobald` instance
+
+    .. code-block:: bash
+
+        $ systemctl status cobald@instance-name
+
+ -  :py:mod:`cobald` instance starts at system boot
```
Comment (eileen-kuehn) [inline docs/source/daemon/systemd.rst]:
```suggestion
 -  enable :py:mod:`cobald` instance start at boot time
```

**Comment by maxfischer2781** [comment]
@mschnepf @eileen-kuehn is this ready for merging? Looks good to me.

**Pull request merged**